### ISSUE TITLE
JavaScript/ Web examples

### ISSUE THREAD

**Issue by BenRacicot** [WalletConnect/web-examples#45]
Hi all!
I noticed that currently there are only React-based examples here as well as the V2 startup pages. 

There is a Node.js section but that combined with the examples here allude to WalletConnect may not be compatible with native JS or JS frameworks outside of React.

I'm an Angular-specific dev but native JS and TS can be applied directly within that framework.

Please let me know how I can best help.

**Comment by EvilJordan** [comment]
Just commenting here about this as well. Does v2 even work without React?

**Comment by BenRacicot** [comment]
Yeah it works, but there is zero developer experience points here. I've even offered to help write the docs and the leads ignored me. 
I did get the project to work through beta to latest but man it was an enormous pain and I'm not going to continue.
